src/Components/AsideMenu/Accordion/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './index.scss';
+class Accordion extends Component {
+  constructor() {
+    super();
+    this.state = { isShowList: false };
+  }
+
+  onShowList = (e) => {
```

Comment:
```suggestion
  toggleList = (e) => {
```
on은 타이밍에 관한 접두어라서 좀 더 기능이 반영된 toggle이라는 네이밍 제안합니다

Reply:
수정했습니다.